lib/takeaway.rb

```diff
@@ -0,0 +1,14 @@
+class Takeaway
+
+  def initialize(menu = @menu)
```

Comment:
Could you create your list of dishes in initialise as an array or hash? Good explanation of your thought process in the readme though.